src/Persistence/PaYaml/Models/SchemaV3/PaModule.cs

```diff
@@ -10,6 +10,8 @@ public record PaModule
 {
     public AppInstance? App { get; init; }
 
+    public EditorStateInstance? EditorState { get; init; }
```

Comment:
maybe we should put this after all the known object types? as the last property.